shared/constants/app-state.ts

```diff
@@ -41,3 +43,12 @@ export enum PasswordChangeToastType {
   Success = 'success',
   Errored = 'errored',
 }
+
+export type NetworkConnectionBanner =
```

Comment:
This may not be the right place for this constant. But it does need to be shared considering we access it both in AppStateController and in the UI.